bin/configs/java-jersey2-8-oneOfMixed.yaml

```diff
@@ -0,0 +1,7 @@
+generatorName: java
+outputDir: samples/client/others/java/jersey2-oneOf-Mixed
```

Comment:
can you please add the new sample folders to `.github/workflows/samples-java-client-jdk11.yaml` so that CI will test it moving forward?

Reply:
Done.